test/relayclient/KnownRelaysManager.test.ts

```diff
@@ -11,28 +11,34 @@ import {
   TestPaymasterConfigurableMisbehaviorInstance,
   TestRecipientInstance
 } from '../../types/truffle-contracts'
-import { evmMineMany } from '../TestUtils'
+import { evmMineMany, startRelay, stopRelay } from '../TestUtils'
 import { prepareTransaction } from './RelayProvider.test'
-import RelayRegisteredEventInfo from '../../src/relayclient/types/RelayRegisteredEventInfo'
+import HttpClient from '../../src/relayclient/HttpClient'
+import HttpWrapper from '../../src/relayclient/HttpWrapper'
+import sinon from 'sinon'
+import { ChildProcessWithoutNullStreams } from 'child_process'
+import { RelayRegisteredEventInfo } from '../../src/relayclient/types/RelayRegisteredEventInfo'
 
 const RelayHub = artifacts.require('RelayHub')
 const StakeManager = artifacts.require('StakeManager')
 const TestRecipient = artifacts.require('TestRecipient')
 const TestPaymasterConfigurableMisbehavior = artifacts.require('TestPaymasterConfigurableMisbehavior')
 
-async function stake (stakeManager: StakeManagerInstance, relayHub: RelayHubInstance, manager1: string, owner: string): Promise<void> {
-  await stakeManager.stakeForAddress(manager1, 1000, {
+export async function stake (stakeManager: StakeManagerInstance, relayHub: RelayHubInstance, manager: string, owner: string): Promise<void> {
+  await stakeManager.stakeForAddress(manager, 1000, {
     value: ether('1'),
     from: owner
   })
-  await stakeManager.authorizeHub(manager1, relayHub.address, { from: owner })
+  await stakeManager.authorizeHub(manager, relayHub.address, { from: owner })
 }
 
-async function register (stakeManager: StakeManagerInstance, relayHub: RelayHubInstance, manager: string, worker: string, url: string): Promise<void> {
+export async function register (stakeManager: StakeManagerInstance, relayHub: RelayHubInstance, manager: string, worker: string, url: string, baseRelayFee?: string, pctRelayFee?: string): Promise<void> {
```

Comment:
nitpick: you don't use the stakeManager, so why have it as parameter?

Reply:
will fix on next commit